package.json

```diff
@@ -39,19 +39,19 @@
     "@guardian/prettier": "1.0.0",
     "@semantic-release/github": "8.0.4",
     "@types/jest": "27.4.1",
-    "@types/node": "16.11.38",
+    "@types/node": "17.0.42",
```

Comment:
Should this remain at 16.x.x as the project is using Node 16?

Reply:
yeah i think so

Reply:
pretty sure "yes"